src/main/kotlin/org/neo4j/graphql/Translator.kt

```diff
@@ -554,18 +550,19 @@ class Translator(val schema: GraphQLSchema) {
 
 
 object SchemaBuilder {
```

Comment:
we should probably pull it out of the translator file

Reply:
See PR #50 coolio

Reply:
done in the next PR